Core/src/main/java/org/tribuo/transform/TransformTrainer.java

```diff
@@ -81,10 +86,24 @@ public TransformTrainer(Trainer<T> innerTrainer, TransformationMap transformatio
 
     @Override
     public TransformedModel<T> train(Dataset<T> examples, Map<String, Provenance> instanceProvenance) {
+        
+        logger.info(String.format("%s %s", TransformTrainer.class.getName(), logger.getLevel()));
+        if(logger.isLoggable(Level.FINE)) {
+            logger.info(String.format("It says fine is loggable?"));
+        }
+        logger.fine(String.format("Creating transformers"));
+        logger.fine("Really doing it, though. Like for real");
```

Comment:
I don't think this logging statement needs to be here.

Reply:
There were actually a few that didn't need to be there, I took them all out, including the flush in the next one.